docs/guides/advanced/images.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Images & files
+---
+
+# Working with images and files
+
+You can upload images and files to a Coda doc, including in a table using dedicated column types. Packs likewise have the ability to work with binary data.
+
+[View Sample Code][samples]{ .md-button }
+
+
+## Parameters and result types
+
+Formulas can accept an image as a parameter using the parameter type `Image`, which is then passed to the `execute` function as a URL. Likewise, formulas can return images as URLs, using the hint type `ImageReference` or `ImageAttachment`. When using an image reference the image is always loaded from the source URL, while for attachments Coda copies the image from the source URL into the document and shows that local copy. Image attachments should be used in most cases, but an image reference may make more sense if you expect the image to be updated often and want to ensure the doc is always using the latest image.
```

Comment:
Hmm how common is it that an image at the same url will update frequently? Your browser would also have to not cache it. The main reason we've had so far for hotlinking is that we're contractually obligated to (Unsplash).

Reply:
Oh, interesting. I was thinking of something like those "build: passing" badge images you see on GitHub, or a company's icon. I've add a note about the contractual part as well.